analytics.go

```diff
@@ -57,7 +57,7 @@ type client struct {
 
 	// This HTTP client is used to send requests to the backend, it uses the
 	// HTTP transport provided in the configuration.
-	http http.Client
+	http *http.Client
 }
```

Comment:
There's no hard rule on whether a program should use a pointer or a value, the language automatically takes the address for methods with pointer receivers ;)